pkgs/applications/version-management/pijul/default.nix

```diff
@@ -4,16 +4,16 @@ with rustPlatform;
 
 buildRustPackage rec {
   name = "pijul-${version}";
-  version = "0.10.0";
+  version = "0.11.0";
 
   src = fetchurl {
-    url = "https://pijul.org/releases/${name}.tar.gz";
-    sha256 = "1lkipcp83rfsj9yqddvb46dmqdf2ch9njwvjv8f3g91rmfjcngys";
+    url = "https://pijul.org/releases/pijul-0.11.0.tar.gz";
+    sha256 = "e60793ab124e9054c1d5509698acbae507ebb2fab5364d964067bc9ae8b6b5e5";
   };
 
-  cargoPatches = [
-    ./libpijul.patch
-  ];
+  # cargoPatches = [
+  #   ./libpijul.patch
+  # ];
 
```

Comment:
If the patch is no longer needed, these lines and the file should be removed.

Reply:
removed